README.md

```diff
@@ -1,18 +1,25 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # FINAL CAPSTONE (FRONT END)
+In this project, we created a website for renting exotic cars (exo-cars). It fetches data from an API we also created and deployed on Heroku. It allows users to register with token validation, see the available cars and make reservations. Admins are allowed to add and remove cars from the database. 
+
+<img src="./public/assets/images/homepage.png" alt="homepage" width="300" style="border: black solid 1px"/>
+<img src="./public/assets/images/detailspage.png" alt="detailspage" width="300" style="border: black solid 1px"/>
 
 ## Built With
 
 - REACT JS.
 - REACT ROUTER.
 - REDUX TOOLKIT.
+- JEST
+### Live demo
 
-### Setup
+[EXO-CARS](https://exo-cars.herokuapp.com)
 
 ### Install
 
 To view this document on your local computer:
+
```

Comment:
- [ ] In the "Run tests" section of your readme file, the command should be "npm run test", instead of just "Jest".   That way, your users will be able to run the tests. 💪